### ISSUE TITLE
Create tag 0.1.1

### ISSUE THREAD

**Issue by renemarc** [titilambert/pyebox#5]
Thanks for this useful library and for the Home Assistant component you wrote! 🍻 😃 

I've patched my Home Assistant installation to support Ebox's "unlimited" data packages by pointing the dependency declaration for [pyebox](https://github.com/titilambert/pyebox) to commit [741d7be](https://github.com/titilambert/pyebox/tree/741d7be7ddb0e2c641877fb43b860a2fbfd11d5e) on your repo. It works, but is kind of a dirty fix. Simply pointing the Home Assistant component to a stable release tag for instance yields no error, but returns blank states.

Until the component is fixed to work with the asynchronicity of [aiohttp](https://github.com/aio-libs/aiohttp), what would be best is to have a tag between 0.1.0 that still uses [requests](https://pypi.python.org/pypi/requests) and the later stable release tags that fixed the unlimited issue but also switched to using aiohttp. This way a simple PR over at Home Assistant to bump the pyebox requirements to 0.1.1 would make your component automatically work again for folks who signed up for unlimited Ebox data packages.

TL;DR: tagging commit 741d7be as 0.1.1 would do. Thoughts?

**Comment by renemarc** [comment]
Closing this issue, it's outdated and no longer relevant.
